go.mod

```diff
@@ -3,16 +3,28 @@ module github.com/ipfs/ipfs-ds-convert
 go 1.12
```

Comment:
Can you also add in a commit that bumps the minimum go version to 1.14 (to match what's in CI and go-ipfs)?